gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/test/java/io/gravitee/am/gateway/handler/scim/resources/users/UpdateUserEndpointHandlerTest.java

```diff
@@ -210,6 +213,24 @@ public void shouldUseASelectedIdp() throws Exception {
                 "OK", null);
     }
 
+    @Test
+    public void shouldAcceptCustomGraviteeUser() throws Exception {
```

Comment:
After the `testRequest`, we should test if the `userService.update` receive a user parameter with the right additionalInformation.

Reply:
FP